### PR TITLE
Port scripts to support both Python 2 and 3

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by chuan-wang** [comment]
Validation 21_11 has been approved. /CW

File: README.md [hunk 1]
```diff
@@ -267,11 +267,9 @@ zendesk_attachment_backup.py -i ngisweden-yyyymmdd/tickets.xml
 ```
 
 ###### Usage
-If you're using this on `tools` for the first time, you'll need to set up conda.
-`tools` only has v2.6 of Python installed by default, which is old and not
-compatible with this script
+The scripts are compatible with Python 2.7+ and Python 3.
 
-These instructions get a copy of Python 2.7 for you. You only need to do this once:
+Instructions in below show how to get a copy of Python 2.7. You only need to do this once:
```
Comment (aanil) [inline README.md:272]:
```suggestion
The instructions below show how to get a copy of Python 2.7. You only need to do this once:
```

**Review by aanil — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (chuan-wang) [reply README.md:272]: Fixed as suggested. /CW

**Pull request merged**